Modules/ZDC/include/ZDC/ZDCRawDataTask.h

```diff
@@ -56,6 +56,26 @@ class ZDCRawDataTask final : public TaskInterface
     TH2* histo;
     std::vector<std::string> condHisto;
```

Comment:
This class has become huge. I don't think that it is manageable, nor that it allows to really understand what is going on. I encourage you to think about some refactoring, in the future.

Reply:
you're right, now I have to finish adding some plots for VDM commissioning. Then I'll try to see how to better organize the code. Thanks for the advice